org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/InitializeLaunchConfigurations.java

```diff
@@ -58,6 +58,13 @@ public static String getVSCodeLocation(String appendPathSuffix) {
 	}
 
 	public static String getNodeJsLocation() {
+		{
+			String nodeJsLocation = System.getProperty("NodeJsLocation");
```

Comment:
We should use a more explicit property name, which mentions a namespace for clarification of what it impacts and what may read it. I suggest something like `org.eclipse.wildwebdeveloper.nodeJSLocation`.

Reply:
Ok I will use `org.eclipse.wildwebdeveloper.nodeJSLocation`. I am waiting for the ECA approval email. I have very little experience in team collaboration with Git and Github, I usually do the modifications in the same Github website. I have to dedicate some time to learn how all these patches and pull requests work. Thanks.